src/SldStyleParser.ts

```diff
@@ -65,6 +66,8 @@ export type ConstructorParams = {
   symbolizerUnits?: string;
   parserOptions?: X2jOptionsOptional;
   builderOptions?: XmlBuilderOptionsOptional;
+  translations?: Partial<SldStyleParserTranslations>;
```

Comment:
If you use `Partial` you usually loose some informations and typings get less strict. Maybe you can just make the Locale keys optional too.